smoke/vrf_forward_frr_test.sh

```
@@ -0,0 +1,52 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2025 Maxime Leroy, Free Mobile
+
+. $(dirname $0)/_init_frr.sh
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)